app/src/main/res/xml/preferences.xml

```diff
@@ -1,29 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
-
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+<PreferenceScreen
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    >
 
     <ListPreference
         android:key="defaultLicense"
         android:title="@string/preference_license"
-        android:defaultValue="CC BY-SA"
+        android:entries="@array/pref_defaultLicense_entries"
+        android:entryValues="@array/pref_defaultLicense_values"
+        android:defaultValue="@string/license_name_cc_by_sa_4_0"
         />
+
     <CheckBoxPreference
+        android:key="allowGps"
         android:title="@string/allow_gps"
         android:defaultValue="false"
         android:summary="@string/allow_gps_summary"
-        android:key="allowGps" />
-
-    <Preference android:title="@string/become_a_tester_title"
-        android:summary="@string/become_a_tester_description">
-        <intent android:action="android.intent.action.VIEW"
-            android:data="@string/beta_opt_in_link" />
-    </Preference>
+        />
 
     <CheckBoxPreference
         android:title="@string/preference_theme"
         android:defaultValue="true"
         android:summary="@string/preference_theme_summary"
         android:key="theme" />
 
+    <Preference android:title="@string/become_a_tester_title"
```

Comment:
I don't think modifying the order of the preferences is related to this PR. As a side note, some of the code styling changes in this file can make it hard to review the PR.